compose/service.py

```diff
@@ -657,7 +657,7 @@ def _get_container_host_config(self, override_options, one_off=False):
         cap_add = options.get('cap_add', None)
         cap_drop = options.get('cap_drop', None)
         log_config = LogConfig(
-            type=options.get('log_driver', 'json-file'),
+            type=options.get('log_driver', ""),
             config=options.get('log_opt', None)
         )
```

Comment:
Just noticed there was no test added to verify the bug in #1863 was fixed by this change